src/components/Folder/Folder.tsx

```diff
@@ -87,23 +86,13 @@ export interface FolderProps<T, P = unknown> {
   onFileUpload?: (parentFolderId: string) => void;
   maxDepth?: number;
   highlightTemporaryFolders?: boolean;
+  withBorderHighlight?: boolean;
 }
 
-const getClassForFolderName = (
-  highlightTemporaryFolders: boolean | undefined,
-  currentFolder: FolderInterface,
-  highlightedFolders: string[] | undefined,
-) => {
-  if (highlightTemporaryFolders && !currentFolder.temporary) {
-    return 'text-secondary';
-  }
-  if (highlightTemporaryFolders && currentFolder.temporary) {
-    return 'text-primary';
-  }
-
-  return highlightedFolders?.includes(currentFolder.id)
-    ? 'text-accent-primary'
-    : 'text-primary';
+const hoverIconClasses = {
+  [FeatureType.Chat]: 'hover:text-accent-secondary',
+  [FeatureType.Prompt]: 'hover:text-accent-tertiary',
+  default: 'hover:text-accent-primary',
```

Comment:
we have overridind and `text-accent-secondary` and `text-accent-tertiary` will apply automatically, but not directly
![image](https://github.com/epam/ai-dial-chat/assets/20163971/d13c0c61-4a56-4db2-aab5-e796b121179e)
i.e. you should use only `text-accent-primary`